src/platform/graphics/graphics-device-create.js

```diff
@@ -107,5 +107,5 @@ function createGraphicsDevice(canvas, options = {}) {
         next();
     });
 }
-
+export default createGraphicsDevice;
```

Comment:
is this line intentional?

Reply:
Oh good spot. Unintentionally left over from some previous testing

Reply:
done 👍